tests_v2/test_browser_goto_split.py

```
@@ -26,6 +30,79 @@ def focused_pane_id(client: cmux) -> Optional[str]:
+
+def _wait_url_loaded(client: cmux, browser_id: str, timeout_s: float = 10.0) -> None:
+    """Poll the browser surface until it reports a non-blank loaded URL.
+
+    Replaces a blind time.sleep page-load wait; returns as soon as the page URL
+    is observable, and only spends the full deadline on the failure path.
+    """
+    deadline = time.time() + timeout_s
+    while time.time() < deadline:
+        try:
+            url = client.get_url(browser_id)
+        except Exception:
```

Ruff (0.15.17)

[warning] Do not catch blind exception: `Exception`

(BLE001)